index.js

```diff
@@ -56,7 +57,11 @@ function openFile (file, sonic) {
 
     // start
     if (!sonic._writing && sonic._len > sonic.minLength && !sonic.destroyed) {
-      actualWrite(sonic)
+      if (sonic.contentMode === 'buffer') {
+        actualWriteBuffer(sonic)
+      } else {
+        actualWrite(sonic)
+      }
```

Comment:
Can we call `_actualWrite` instead of having an `if` here?

Reply:
if we assign it as `sonic._actualWrite` I think it's possible to do so. At the same time these are not hot paths, so not sure if it's worth it

Reply:
Oh, I was thinking of using `_actualWrite` for maintainability rather than performance